### ISSUE TITLE
Imputation of LCFS onto FRS

### ISSUE THREAD

**Issue by nikhilwoodruff** [UBICenter/uk-carbon-dividend#2]
After the LCFS openfisca-uk-data dataset is complete, we should plan out how best to impute LCFS model variables onto the FRS dataset. It looks like the best way to do this is using the random forest-based imputation with Beta distribution matching of aggregates from `rf_impute` in `synthimpute`.

**Comment by MaxGhenis** [comment]
For this project, we'd only need to impute carbon emissions right?

**Comment by nikhilwoodruff** [comment]
Yeah, I think so. Though an alternative might be imputing the expenditure on the different categories, and calculating carbon emissions within openfisca using parameters - an advantage of this is that it's then a lot simpler to later come back to when we add VAT, but probably overcomplicated for this project?
